src/Notifications/NotificationCenterController.cs

```diff
@@ -91,6 +99,35 @@ private void NotificationsButton_Click(object sender, RoutedEventArgs e)
             notificationUIPopup.IsOpen = !notificationUIPopup.IsOpen;
             if (notificationUIPopup.IsOpen)
                 notificationUIPopup.webView.Focus();
-        }        
+        }
+
+        private string FetchNotificationURL()
```

Comment:
Is there a similar function on package manager side? If there is, I expect us to re-use the function. Maybe we can put this as a shared utility function

Reply:
There isn't I created this, but I can try to put it in utility and use it on both locations.

Reply:
If there isn't, would you find out how we read the endpoint value for the package manager? Would love to align

Reply:
It is the same code: https://github.com/DynamoDS/Dynamo/blob/20851b250a91203bab7275d78bc60b34982a7bcf/src/DynamoPackages/PackageManagerExtension.cs#L123
but without a method

Reply:
@zeusongit Gotcha 👍🏻